crawler.py

```diff
@@ -328,7 +328,8 @@ def start_driver(self):
 
             # loads parallel extension to run alongside pb
             if self.load_extension:
-                self.driver.install_addon(self.load_extension)
+                parallel_addon_url = os.path.join('~', self.load_extension)
```

Comment:
Does this assume that your current working directory is your home directory? (Does this work in Docker?) This doesn't seem like a safe assumption.

Reply:
This works in Docker for me. Is there a safer alternative you'd recommend?

Reply:
Re: our conversation earlier today -- I updated the most recent commit to make any path absolute that's passed to a firefox scan